addon/components/bulma-radio.js

```diff
@@ -12,16 +12,21 @@ export default BulmaInput.extend({
 
   classNames: ['radio'],
   type: 'radio',
+
   // Bindings are not comprehensive. More complex implementations should use a native element with classes applied
   classNameBindings: [
```

Comment:
@jbailey4 I think this was a mistake. It seems like this should have been `attributeBindings ` instead of `classNameBindings`. May not assign a copy below.